vignettes/getting-started.Rmd

```diff
@@ -305,6 +337,8 @@ run_log(MODEL_DIR, .include = "2 compartment") %>%
 ```{r cleanup bottom, include=FALSE}
 # delete old files
 cleanup()
+# remove new tag from mod1
+mod1 <- mod1 %>% remove_tags("orig acop model")
```

Comment:
FYI this change was to revert the new tag that gets added to mod1. Probably should have been there before, but wouldnt impact anything unless it was rendered locally/the changes were committed.